src/locale/locales/en/common.json

```diff
@@ -315,5 +315,9 @@
     "importAccountManually": "Import account manually",
     "importAccountManuallyDescription": "If you think you have an account that is not showing up, you can import it manually. Just enter the account you want to import.",
     "enterTheAccount": "Enter the account",
-    "swap": "swap"
+    "swap": "swap",
+    "addNear": "add near",
+    "buyNear": "Buy NEAR",
+    "depositNearFromOtherWallet": "Deposit NEAR from other wallet",
+    "iNeedSupport": "I need support"
```

Comment:
Faltan locales again! :vietnam: :it: ...

Reply:
Done